test/spec.js

```diff
@@ -12,7 +12,31 @@ describe('samay.js test suite', function(){
 	});
 
 	it('format test YYYYMMDD against format MM/DD/YYYY HH:mm', function() {
-		expect(samay('20180330', 'YYYYMMDD').format('MM/DD/YYYY HH:mm')).toBe('03/30/2018 00:00')
+		expect(samay('20180228', 'YYYYMMDD').format('MM/DD/YYYY HH:mm')).toBe('02/28/2018 00:00')
+	});
+
+	it('format test YYYYMMDD against format MM/DD/YYYY HH:mm', function() {
+		expect(samay('20190228', 'YYYYMMDD').format('MM/DD/YYYY HH:mm')).toBe('02/28/2019 00:00')
+	});
+
+	it('format test YYYYMMDD against format MM/DD/YYYY HH:mm', function() {
+		expect(samay('20200229', 'YYYYMMDD').format('MM/DD/YYYY HH:mm')).toBe('02/29/2020 00:00')
```

Comment:
Test suite is failing in this test case, not sure why that is happening:

<img width="839" alt="Screenshot 2019-03-31 at 2 09 34 AM" src="https://user-images.githubusercontent.com/3415488/55281445-795fe300-535a-11e9-9da5-e3bf9b3f69af.png">